src/bci_build/package/appcontainers.py

```diff
@@ -129,6 +129,9 @@
     for os_version in ALL_NONBASE_OS_VERSIONS
 ]
 
+_MARIADB_README = (
```

Comment:
Is this even required? If I understood your code correctly, then the dockerfile generator should discover the readme template by itself

Reply:
Yes, the file is called `README.server.md.j2`, the magic works if the file is called `README.md.j2`.

We have a server and client MariaDB image, the idea for this name is that I plan to create a `README.client.md.j2` and keep both readmes in the same folder.

I would not object to creating a new folder for mariadb-client and both file names could be `README.md.j2` if you prefer.

Reply:
how about simply an if condition inside the `README.md.j2` that goes between client or server?

that seems much simpler to me. Also if we want to share content, we should simply include it from the  shared templates location.

Reply:
I considered that, but both READMES would be so far apart that I decided that two templates would be better. I'll give it a try and see how it looks.